src/main/java/com/bakdata/common_kafka_streams/util/ErrorCapturingValueTransformerWithKey.java

```diff
@@ -0,0 +1,63 @@
+package com.bakdata.common_kafka_streams.util;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.streams.kstream.ValueTransformerWithKey;
+import org.apache.kafka.streams.processor.ProcessorContext;
+
+@Slf4j
+@RequiredArgsConstructor
+public class ErrorCapturingValueTransformerWithKey<K, V, VR>
+        implements ValueTransformerWithKey<K, V, ProcessedValue<V, VR>> {
+    private final @NonNull ValueTransformerWithKey<? super K, ? super V, ? extends VR> wrapped;
+
+    /**
+     * Wrap a {@code ValueTransformerWithKey} and capture thrown exceptions.
+     * <pre>{@code
+     * final ValueTransformerWithKey<K, V, VR> transformer = ...;
+     * final KStream<K, V> input = ...;
+     * final KStream<K, ProcessedValue<V, VR>> processed = input.transformValues(() -> captureErrors(transformer));
```

Comment:
This is a bad example, since you shouldn't reuse the Transformer object. 
Reusing a transformer object may lead to a NullPointerException as described on [here on StackOverflow](https://stackoverflow.com/questions/39067846/periodic-npe-in-kafka-streams-processor-context).

(You may have similar issues in Javadocs in other transformer classes)